doc/release-notes/release-7.sgml.in

```diff
@@ -30,7 +30,7 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 
 <p>The most important of these new features are:
 <itemize>
-	<item>
+	<item> cache manager non_peers report removal
```

Comment:
I removed this change in branch commit 592887c because I do not think this PR changes qualify to be listed in the "most important new features" subsection of the "Major new features" section. However, if others insist on its presence, I will restore it.

Reply:
Please revert. This removal **is** a major change for users paying any kind of attention to these reports.